Ginger/GingerCoreNET/Run/GingerRemoteExecutionUtils.cs

```diff
@@ -159,5 +159,20 @@ public static string GetReportHTMLServiceUrl()
             }
             return baseURI;
         }
-       }
+
+        public static string GetOnlineHTMLReportlink(Guid? executionGuid)
+        {
+            var htmlServiceUrl = GetReportHTMLServiceUrl();
+            if (!string.IsNullOrEmpty(htmlServiceUrl))
+            {
+                return htmlServiceUrl + "?ExecutionId=" + executionGuid;
+            }
+            else
+            {
+                Reporter.ToUser(eUserMsgKey.StaticErrorMessage, "Centralized Html Report Service URL is missing in General Report Configurations.\nPlease go to Configurations > Reports > Execution Logger Configurations to configure the HTML Report URL");
```

Comment:
Add it to the log instead of showing it to the user cause in case of unattended execution, user won't have the popup

Reply:
done